doc/content/gettingStarted/traffic.md

```diff
@@ -0,0 +1,181 @@
+---
+title: "Device interaction"
```

Comment:
Suggesting

>Data and Events

Reply:
Working but doesn't use the infinitive form as the other titles.
`Connect to Data and Events` ?